.travis.yml

```diff
@@ -6,7 +6,7 @@ branches:
   only:
     - main
     - feature-terra-functional-testing
-    - terra-funtional-testing-part-1
+    - functional-testing-part-2
```

Comment:
My memory is fuzzy on this. Does this need to be the development branch or only the branch being merged into? If you're merging into  `feature-terra-functional-testing` and it's already listed do you need to list your story branch?

Reply:
Circling back on this. The issue is that the main branch `feature-terra-functional-testing` has a typo.It's actually `feature-terra-funtional-testing`

Reply:
Oops, thanks